pkg/apis/harvester.cattle.io/v1alpha1/template.go

```diff
@@ -33,25 +49,27 @@ type VirtualMachineTemplateVersion struct {
 	Status VirtualMachineTemplateVersionStatus `json:"status,omitempty"`
 }
 
-type VirtualMachineTemplateSpec struct {
-	Description      string `json:"description,omitempty"`
-	DefaultVersionID string `json:"defaultVersionId,omitempty"`
-}
+type VirtualMachineTemplateVersionSpec struct {
```

Comment:
would prefer to add shortName as `vmtemplateversion` and print its templateId and status.version.

Reply:
Updated